test/wpt/status/streams.json

```diff
@@ -16,5 +16,8 @@
   },
   "readable-byte-streams/non-transferable-buffers.any.js": {
     "fail": "Needs more investigation"
+  },
+  "readable-byte-streams/general.any.js": {
+    "fail": "Web Platform Tests fail to consider platform endianness"
```

Comment:
Shoot, you're right. We're [writing to a `Uint8Array`](https://github.com/web-platform-tests/wpt/blob/a658b79cd0d686edf6e5eb718a0e55e9a3933e67/streams/readable-byte-streams/general.any.js#L940-L942) and [reading it back from a `Uint16Array`](https://github.com/web-platform-tests/wpt/blob/a658b79cd0d686edf6e5eb718a0e55e9a3933e67/streams/readable-byte-streams/general.any.js#L960), incorrectly assuming that we're on a little-endian platform. I think I spotted that a while ago, but I forgot to make an issue about it. I'll look into it.

Reply:
I've at least manually verified that the implementation is doing the right thing here. I will revisit this after the WPT's are updated.

Reply:
web-platform-tests/wpt#29488 has landed, you should now be able to re-enable these tests. 🤞

Reply:
That was fast! Thank you!